### DISCUSSION TITLE
EmbedArt: mode to automatically clear embedded art

### DISCUSSION THREAD

**Discussion by nagisa** [beetbox/beets#5033] [Ideas]
The current `EmbedArt` plugin currently appears to have configuration to only automatically run or not to run the `embedart` subcommand on an import. However this plugin also provides a subcommand to remove the embedded art – `clearart`. It would be great if there was a way to specify that the embedded art should be automatically cleared upon an import.

**Comment by DjSlash** [comment]
You can use the [zero plugin](https://beets.readthedocs.io/en/stable/plugins/zero.html) instead for removing the embedded art. Example:

```yaml
zero:
    fields: images
```
